src/lib/pages/backoffice/Document/DocumentDetails/DocumentPurchaseOrders/DocumentPurchaseOrders.js

```diff
@@ -0,0 +1,112 @@
+
+import { dateFormatter } from '@api/date';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withCancel } from '@api/utils';
+import { orderApi } from '@api/acquisition';
+import { ResultsTable } from '@components/ResultsTable/ResultsTable';
+import { Loader } from '@components/Loader';
+import { Link } from 'react-router-dom';
+import { AcquisitionRoutes } from '@routes/urls';
+
+export default class DocumentPurchaseOrders extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: false,
+      orders: {},
+      isLoading: false
+    };
+    const {
+      documentDetails: { pid },
+    } = this.props;
+    this.fetchPurchaseOrders(pid)
```

Comment:
This should be don in the [componentDidMount](https://reactjs.org/docs/react-component.html#componentdidmount) hook, not in the constructor.
If you see any other places where this is done, please fix it there too! :)